File: C++/OOP Basic/Q3.cpp
#include <iostream>
using namespace std;

class Student{
	public:
	static int ID;
	string name;

	void diplay(){
		cout<<name<<endl;
			
		}
		static void Id(){
			cout<<ID<<endl;
		}
	
};

int Student :: ID=2;

int main (){
	Student S1,S2;
	S1.name="sara";
	S2.name="Ali";
	S1.diplay();
	S1.Id();
	S2.ID=9;
	S2.diplay();
	S2.Id();
}

File: Python/NLP/Q3.py
#Stop words
import nltk
from nltk.corpus import stopwords
from nltk.tokenize import word_tokenize

#nltk.download("punkt") already downloaded
nltk.download("stopwords")

text="Hi, My name is Sara Ali. I am 19 years old. It felt nice meeting you"
word_tk=word_tokenize(text)

stop_words=set(stopwords.words("english"))

fil_words=[word for word in word_tk if word.lower() not in stop_words]
print(f"Original words: {word_tk}\nAfter removing stop words: {fil_words}")

#Output
#Original words: ['Hi', ',', 'My', 'name', 'is', 'Sara', 'Ali', '.', 'I', 'am', '19', 'years', 'old', '.', 'It', 'felt', 'nice', 'meeting', 'you']
#After removing stop words: ['Hi', ',', 'name', 'Sara', 'Ali', '.', '19', 'years', 'old', '.', 'felt', 'nice', 'meeting']

